starter-code/src/viking.js

```diff
@@ -1,8 +1,40 @@
 // Soldier
-function Soldier() {}
+function Soldier(health,strength){
+    this.health = health;
+    this.strength = strength;
+  
+  }
 
+  Soldier.prototype.attack = function(){
+    return this.strength;
+  }
+  Soldier.prototype.receiveDamage = function(Damage){
+  this.health = this.health - Damage;
+  }
+  
+    
 // Viking
-function Viking() {}
+function Viking(name,health,strength) {
+    Soldier.call(this, name, owner);
+    this.name = name;
+};
+
+function attack(){
```

Comment:
Revisad el uso del `this` y cómo añadir métodos al prototipos de la learning unit ;)